site/docs/v1/tech/apis/_webhook-search-request-parameters.adoc

```diff
@@ -0,0 +1,38 @@
+
+// parameter_prefix is either blank for parameters or "search." for body
+
+[.api]
+
+[field]#{parameter_prefix}description# [type]#[String]# [optional]#Optional#::
+The case-insensitive string to search for in the Webhook description or URL. This can contain wildcards using the asterisk character (`*`). If no wildcards are present, the search criteria will be interpreted as `pass:[*value*]`. If this field is omitted, no filtering will be applied to the `description` or `url` fields.
+
+[field]#{parameter_prefix}numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#{parameter_prefix}orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `name ASC`#::
+The field to order the search results as well as an order direction.
++
+The possible values are:
++
+ * `description` - the description of the Webhook
+ * `id` - the UUID of the Webhook
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Webhook was created
+ * `url` - the Webhook URL
++
+The order direction  is optional. Possible values of the order direction are `ASC` or `DESC`. If omitted, the default sort order is `ASC`.
++
+For example, to order the results by the insert instant in a descending order, use `insertInstant DESC`.
+
+[field]#{parameter_prefix}startRow# [type]#[Integer]# [optional]#Optional# [default]#defaults to `0`#::
+The offset into the total results. In order to paginate the results, increment this value by the [field]#numberOfResults# for subsequent requests.
+
+[field]#{parameter_prefix}tenantId# [type]#[UUID]# [optional]#Optional#::
+Restricts the results to Webhooks associated with the given Tenant, including global Webhooks.
```

Comment:
Does this mean that whatever tenantId I provide, global webhooks will be returned? If so, maybe:

```suggestion
Restricts the results to Webhooks associated with the given Tenant and all global Webhooks.
```